spk/python/Makefile

```diff
@@ -1,7 +1,7 @@
 SPK_NAME = python
 SPK_SHORT_VERS = 2.7
 SPK_VERS = $(SPK_SHORT_VERS).14
-SPK_REV = 19
+SPK_REV = 20
```

Comment:
Why updating Python 2.7? borgbackup runs on Python 3, right?

Reply:
borgbackup depends on python3, but I had to upgrade package msgpack-python that it is used by both python and python3 packages.